### PR TITLE
75 - Creation of the area in the db (back)

### LINKED ISSUE #75: Creation of the area in the database on the back-end.
## Related feature
This will record the user's choices on the front end (which we will not deal with in this section). These choices will then be recorded in the database.
## Improvement
_Describe what the improvement of the feature_

## Extra link (Optional)
_Add any other extra links_

## Screenshots (Optional)
_Add any screenshot_

### PR DESCRIPTION
**Pull request by STom6**
# New feature
added the function setIndb, it registers the information it is sent.
Creation of the functions of addition of data for IssStation & openMétéo.
Definition of googleService at the creation of the account.
# Updated feature

# Bug fixed

### CODE REVIEW

**Review by JohanCDev — DISMISSED** [review]
lgtm

File: Server/Services/googleService.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
      * @param {*} req Unnecessary but mandatory for areaLoop.
      * @param {*} res Unnecessary but mandatory for areaLoop.
```
Comment (osvegn) [inline Server/Services/googleService.js]: Unused

File: Server/Services/twitterService.js [hunk 2]
```diff
@@ -18,12 +18,13 @@ const { TwitterApi } = require('twitter-api-v2')
  * @param {*} action is an argument that determines what action should be performed by the function. It 
  * can take the values 'tweet', 'like' or 'retweet'.
  */
-async function carryOutAnAction(req, res, appKey, appSecret, bearer, hashtagOrMessage, action) {
```
Comment (osvegn) [inline Server/Services/twitterService.js:21]: Update the documentation

File: Server/Services/twitterService.js [hunk 3]
```diff
@@ -80,15 +81,15 @@ function GetIdTwitter(uid) {
 * @param {*} res is an object that handles the HTTP response that will be sent to the user.
 */
 
-function doAct(action, hashtagOrMessage, req, res) {
+function doAct(action, hashtagOrMessage, userData) {
```
Comment (osvegn) [inline Server/Services/twitterService.js:81]: Update the documentation

File: Server/Services/twitterService.js [hunk 4]
```diff
@@ -234,11 +235,10 @@ module.exports = {
     * @param {*} req is an object that contains information about the HTTP request that called this function
     * @param {*} res is an object that handles the HTTP response that will be sent to the user.
     */
-    ActionTw: function(action, hashtagOrMessage, uid, req, res) {
+    ActionTw: function(action, hashtagOrMessage, uid) {
```
Comment (osvegn) [inline Server/Services/twitterService.js:233]: Update documentation

**Review by osvegn — CHANGES_REQUESTED** [review]
Some documentation to update

**Review by CedricCORGE — APPROVED** [review]
LGTM

**Review by JohanCDev — APPROVED** [review]
ntr

**Review by osvegn — APPROVED** [review]
Look good

**Pull request merged**